packaging/rpm/microshift.spec

```diff
@@ -484,6 +502,58 @@ cat assets/optional/gateway-api/kustomization.x86_64.yaml >> %{buildroot}/%{_pre
 mkdir -p -m755 %{buildroot}%{_datadir}/microshift/release
 install -p -m644 assets/optional/gateway-api/release-gateway-api-{x86_64,aarch64}.json %{buildroot}%{_datadir}/microshift/release/
 
+# ai-model-serving
+# Currently only x86_64 is supported. Following `ifarch` prevents building aarch64 RPM by not specifying the files for the aarch64 architecture.
+%ifarch x86_64
```

Comment:
This is a long list. Do we need to do this for each directory?
I forgot if there's an option in RPM spec to do file copy recursively including the dir creation.

Reply:
Actually this has nothing to do with the rpm spec, it just runs shell commands, in this particular case it's `install` command and it does not support recursive copying